package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rn-fetch-blob",
-  "version": "0.10.15",
+  "version": "0.10.16",
```

Comment:
I would rather not increment the version in a PR, but rather wait until publish time to change the version number